webofneeds/won-owner-webapp/src/main/webapp/gulpfile.js

```diff
@@ -54,6 +54,18 @@ gulp.task('sass', function(done) {
         .on('end', done);
 });
 
+/**
+ * Copies over static resources, that libraries in
+ * the bundle need to be in a place relativ to
+ * themselves -- and thus the bundle.
+ */
+gulp.task('copy-static-res', function(done) {
+    return gulp.src([
+        './jspm_packages/npm/leaflet@0.7.7/dist/images/**/*'
```

Comment:
is getting this resource in the right place not possible without a static reference to the version number of leaflet? we should avoid this since the error might reoccur when the leaflet version is updated without updating this code